yt_dlp/extractor/youtube.py

```diff
@@ -3973,13 +3974,10 @@ def append_client(*client_names):
             )
 
             require_po_token = self._get_default_ytcfg(client).get('REQUIRE_PO_TOKEN')
-            if not po_token and require_po_token:
+            if not po_token and require_po_token and 'missing_pot' in self._configuration_arg('formats'):
```

Comment:
the formats are still deprioritized even if `missing_pot` is passed, though, right? we are still setting `'preference': -20`

Reply:
yes

Reply:
disregard

Reply:
As I understand this code:

- `missing_pot` wasn't used: the clients requiring pot are skipped. The preference doesn't matter?
- this case, `missing_pot` was used: `deprioritize_pr` is set to `True` and the preference is `-20`

Reply:
> missing_pot wasn't used: the clients requiring pot are skipped. The preference doesn't matter?

The client is not skipped, but the formats requiring PO Token are. 

> this case, missing_pot was used: deprioritize_pr is set to True and the preference is -20

Correct